src/Tasks/Copy.cs

```diff
@@ -664,7 +669,7 @@ private bool ValidateInputs()
             }
 
             // There must be a destination (either files or directory).
-            if (DestinationFiles == null && DestinationFolder == null)
+            if ((DestinationFiles == null || DestinationFiles.Length == 0) && DestinationFolder == null)
```

Comment:
I would prefer to do not have check `DestinationFiles.Length == 0` here: this situation is caught few lines below with more detailed error `TwoVectorsMustHaveSameLength`. I see similarities with the SourceFiles conditions, why, I suppose, this line was changed, but since it anyway would be some error, let's pick the more elaborate one.

Reply:
Makes sense